chain/ethereum/src/ethereum_adapter.rs

```diff
@@ -1604,6 +1679,74 @@ pub(crate) fn parse_block_triggers(
             EthereumBlockTriggerType::Every,
         ));
     }
+
+    match &block_filter.wildcard_filters {
+        Some(WildcardBlockFilter { polling: true, .. }) => {
+            triggers.push(EthereumTrigger::Block(
+                block_ptr3,
+                EthereumBlockTriggerType::Every,
+            ));
+        }
+        None => {
+            let has_polling_trigger =
+                &block_filter
+                    .polling_intervals
+                    .iter()
+                    .any(|(start_block, _, interval)| match interval {
+                        0 => false,
+                        _ => (block_number - *start_block) % *interval == 0,
+                    });
+            if *has_polling_trigger {
+                triggers.push(EthereumTrigger::Block(
+                    block_ptr3,
+                    EthereumBlockTriggerType::Every,
+                ));
+            }
+        }
+        _ => (),
+    }
+    triggers
+}
+
+pub(crate) fn parse_initialization_triggers(
+    block_filter: &EthereumBlockFilter,
+    block: &EthereumBlockWithCalls,
+) -> Vec<EthereumTrigger> {
+    if block_filter.is_empty() {
+        return vec![];
+    }
+
+    let block_ptr = BlockPtr::from(&block.ethereum_block);
+    let block_ptr3 = block_ptr.cheap_clone();
+    let block_number = block_ptr.number;
+
+    let mut triggers = vec![];
+
+    match &block_filter.wildcard_filters {
+        Some(WildcardBlockFilter { once: true, .. }) => triggers.push(EthereumTrigger::Block(
```

Comment:
NOTE: This seems that it can be further optimized. These once triggers are only triggered at the startBlock so this seems very unncessary to send Every triggers in every block